.travis_script.bash

```diff
@@ -41,33 +41,15 @@ ponyc-build-docs(){
 
 case "${TRAVIS_OS_NAME}:${LLVM_CONFIG}" in
 
-  "linux:llvm-config-3.7")
-    ponyc-test
-  ;;
-
-  "linux:llvm-config-3.8")
-    ponyc-test
-  ;;
-
   "linux:llvm-config-3.9")
     # when FAVORITE_CONFIG stops matching part of this case, move this logic
     if [[ "$TRAVIS_BRANCH" == "release" && "$TRAVIS_PULL_REQUEST" == "false" && "$FAVORITE_CONFIG" == "yes" ]]
     then
       ponyc-build-packages
       ponyc-build-docs
-    else
-      ponyc-test
     fi
   ;;
 
-  "linux:llvm-config-4.0")
-    ponyc-test
-  ;;
-
-  "linux:llvm-config-5.0")
-    ponyc-test
-  ;;
-
   "osx:llvm-config-3.7")
```

Comment:
Should we remove osx:llvm-config-3.7 and osx:llvm-config-3.8?

Reply:
They aren't run. I will get as part of another PR.